Sming/Services/SpifFS/spiffs_sming.c

```diff
@@ -1,5 +1,7 @@
 #include "spiffs_sming.h"
 
+DEFINE_PSTR(initFileName, "initialize_fs_header.dat");
```

Comment:
Move that line as one of the fist lines inside the body of `static void spiffs_mount_internal(spiffs_config *cfg)`.

Reply:
This should also be `PSTR_ARRAY(initFileName, "initialize_fs_header.dat")`. (Apologies @kmihaylov I got this wrong!)